aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,149 @@
-.. include:: /../common/stub-topic.txt
+.. _nano-server:
+
+ASP.NET 5 on Nano Server
+========================
+
+By `Sourabh Shirhatti`
+
+.. attention:: This tutorial uses a pre-lease version of the Nano Server installation option of Windows Server Technical Preview 4. You may use the software in the virtual hard disk image only to internally demonstrate and evaluate it. You may not use the software in a live operating environment. Please see https://go.microsoft.com/fwlink/?LinkId=624232 for specific information about the end date for the preview.
+
+In this tutorial, you'll take an existing ASP.NET 5 application and deploy it to a Nano Server instance running IIS.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Introduction
+------------
+
+Nano Server is a new version of Windows Server with a small footprint optimized for the cloud. You can learn more about Nano Server from the official docs `available here <https://msdn.microsoft.com/en-us/library/mt126167.aspx>`_.
+
+In this tutorial, we will using a pre-built Virtual Hard Disk (VHD) for Nano Server which can be downloaded `here <https://msdn.microsoft.com/en-us/virtualization/windowscontainers/nano_eula>`_. This pre-built VHD already includes the Reverse Forwarders and IIS packages which are required for this tutorial.
```

Comment:
> In this tutorial, we will using a pre-built Virtual Hard Disk (VHD) for Nano Server which can be downloaded `here <https://msdn.microsoft.com/en-us/virtualization/windowscontainers/nano_eula>`_. 

In this tutorial, we will using the pre-built `` `Virtual Hard Disk (VHD) for Nano Server <https://msdn.microsoft.com/en-us/virtualization/windowscontainers/nano_eula>_` `` from Windows Server Technical Preview 4.